src/pages/Home.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import StockMaterialsLists from '../components/MatrialList';
+import { filterCompany } from '../components';
+import style from '../style/Home.module.css';
+
+const Home = () => {
+  const navigate = useNavigate();
+  const stockMaterials = useSelector((store) => store.stockMaterials);
+  const [stockMaterial, setStockMaterial] = useState([]);
+
+  useEffect(() => {
+    setStockMaterial(stockMaterials);
+  }, [stockMaterials]);
+
+  const handleSelection = (company) => {
+    const filtered = filterCompany(company, stockMaterials);
+    setStockMaterial(filtered);
+  };
+
+  const handleSearch = (text) => {
+    // eslint-disable-next-line
+    const filteredCompanies = stockMaterials.filter((ex) => ex.companyName.toLowerCase().includes(text));
+    setStockMaterial(filteredCompanies);
+  };
```

Comment:
- [ ] Great job with the design and the functionalities. :+1: There seems to be a bug in the search functionality. The moment I start writing in the text field, every card is gone. Kindly fix it. 🧐️
- [ ] Please remove the line where ESLint is disabled as they help us in achieving a cleaner code. 🤝️

Reply:
thanks for your good and valuable feedback.

Reply:
resolved! ✔️